connector/facet-handler.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+class FacetHandler {
+  static createFacet(workspace, id, data, cb) {
+    function create(next) {
+      workspace.addFacet(id, data, function(err) {
+        next(err);
+      });
+    };
+    function callBack(err, results) {
```

Comment:
`results` not used? And I doubt if you need this `callBack`/`callback` function here and you can replace L14 to:
```
workspace.execute(taskList, cb);
```